### PR TITLE
MCKIN-11065 Fixed empty popups appearing for tips

### PR DESCRIPTION
**Pull request by mudassir-hafeez**
After the first attempt of submitting MRQ feedback, when a user clicks on any item feedback icon, a popup appears with a choice tip. while in the second attempt of resubmitting MRQ feedback, when a user clicks on any item feedback icon after clicking on any choice and before submitting MRQ then a popup appears with an empty choice tip messages.
Please see the picture below for better understanding:
Empty_choice_tip

After this fix, the user will not see the container containing an empty choice tip.

### CODE REVIEW

**Comment by mudassir-hafeez** [comment]
Hi @xitij2000 , can you please review this PR? Actually, the team doesn't have any access for the xblock to review this PR. Thanks

File: setup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 # Constants #########################################################
 
-VERSION = '3.3.9'
+VERSION = '3.4.0'
```
Comment (xitij2000) [inline setup.py]: This can just be a patch bump to 3.3.10

**Comment by xitij2000** [comment]
@mudassir-hafeez Sure. I have scheduled this PR for review next sprint. 

Till then, could you update the ticket description to include more details, and add testing instructions?
Please include tests for this functionality as well.

File: setup.py (hunk 1 above)
Comment (mudassir-hafeez) [reply setup.py]: Addressed

**Comment by mudassir-hafeez** [comment]
@xitij2000, I have updated the test cases and description accordingly. Ping me if you need to update anything else :slightly_smiling_face:

**Comment by Agrendalath** [comment]
Hey @mudassir-hafeez, sorry for the delay, I didn't subscribe to this PR. I'll review it by tomorrow.

**Comment by mudassir-hafeez** [comment]
@Agrendalath thanks for taking this PR to review. I have updated the version 3.3.10 to 3.3.11 as `Kshitij Sobti` had created another version into the xblcok during this delayed period.

**Comment by Agrendalath** [comment]
Thank you for fixing this, @mudassir-hafeez. Looks good to me.

:+1: 
* [x]  I tested this: created MRQ block on local devstack and checked that popups no longer appear after (un)checking answer
* [x]  I read through the code
* [ ]  <s>I checked for accessibility issues</s>
* [ ]  <s>Includes documentation</s>

**Comment by mudassir-hafeez** [comment]
Thanks, @Agrendalath. :slightly_smiling_face: 
Can you please create a release tag against this fix? So I'll be then creating a version bump into the edx-plateform.

**Comment by Agrendalath** [comment]
@mudassir-hafeez, it's already [here](https://github.com/open-craft/problem-builder/releases/tag/v3.3.11) :)

**Comment by mudassir-hafeez** [comment]
> @mudassir-hafeez, it's already [here](https://github.com/open-craft/problem-builder/releases/tag/v3.3.11) :)

Oh great. Thanks, @Agrendalath.

**Pull request merged**